launcher/settingsView/csettingsview_moc.cpp

```diff
@@ -315,10 +345,34 @@ void CSettingsView::on_comboBoxShowIntro_currentIndexChanged(int index)
 	node->Bool() = index;
 }
 
-void CSettingsView::on_comboBoxAutoSave_currentIndexChanged(int index)
+void CSettingsView::on_comboBoxAutosaveMode_currentIndexChanged(int index)
 {
-	Settings node = settings.write["general"]["saveFrequency"];
-	node->Integer() = index;
+	QVariant itemData = ui->comboBoxAutosaveMode->itemData(index, Qt::UserRole);
+	std::string selectedEntry = itemData.toString().toStdString();
```

Comment:
Please, don't store such strings in item data. Instead, create enum and store enum value in item data and them compare with enum instead of string comparison

Reply:
I have started with Enum but Ivan pointed out that there could be problems with translation. I did not understand the remark but in lack of feedback, a quick search on the net showed this itemData as best approach?

I see no problems with it, but if needed  I could revert to previous version that used enums.